app/Http/Controllers/ProductsController_SA.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Products_SA;
+use App\Helpers\ResponseHelper;
+use App\Http\Requests\ProductRequest_SA;
+use App\Http\Resources\ProductResource_SA;
+use App\Helpers\PaginationHelper;
+
+
+class ProductsController_SA extends Controller
+{
+    public function getData(ProductRequest_SA $request)
+    {
+        if (!$request->query()) {
+
+            if ($request->paginate) {
```

Comment:
Simplify the pagination condition check

Reply:
This one also I had simplified but it hasnt't changed here

![image](https://github.com/ZeeshanHabib7/demo-internee-app-git/assets/119349274/3e111d98-55d5-4b5e-ac15-af2e4a213613)